router/middlewares/middlewareLogger.go

```diff
@@ -0,0 +1,39 @@
+package middlewares
+
+import (
+	"net/http"
+
+	"github.com/ditrit/badaas/configuration"
+	"github.com/noirbizarre/gonja"
+	"github.com/noirbizarre/gonja/exec"
+	"go.uber.org/zap"
+)
+
+func buildRequest(template *exec.Template, r *http.Request) string {
```

Comment:
missing documentation, bad naming function, it's better to have `getLogMessage` or something like that